activesupport/lib/active_support/concern.rb

```diff
@@ -137,6 +140,23 @@ def included(base = nil, &block)
       end
     end
 
+    # Define class methods from given block.
+    # You can define multiple +class_methods+ blocks.
```

Comment:
hmm... actually multiple `class_methods` works, but seems it is not by design.
Is it worth to mention about multiple `class_methods`?

Reply:
Actually, I found it interesting that we can have multiple`class_methods` while we cannot have multiple `included`.
Should it raise an exception when we have multiple `class_methods`, for the sake of symmetry?

Reply:
It seems worth discussing (I'd say less worth to disallow things aren't broken though).

Reply:
It might be good to put deprecation warning first when we have multiple `class_methods`.
If so, it's not a good idea to document multiple `class_methods`. What do you think?

Reply:
We don't always describe all behaviors in the API doc (e.g. https://github.com/rails/rails/pull/28490#issuecomment-287804582).
Regardless of whether it is deprecated or not I don't think the multiple `class_methods` is valid use case.

Reply:
Alright, I'll remove multiple case from an example.